tests/test_association_task.py

```diff
@@ -110,9 +190,20 @@ def setUp(self):
 
         bbox = afwGeom.Box2D(self.exposure.getBBox())
         wcs = self.exposure.getWcs()
-        self.expMd = pipeBase.Struct(
-            bbox=bbox,
-            wcs=wcs,)
+
+        ctr_coord = wcs.pixelToSky(bbox.getCenter())
+        max_radius = max(
+            ctr_coord.separation(wcs.pixelToSky(pp))
+            for pp in bbox.getCorners())
+
+        assoc_config = AssociationConfig()
+        assoc_task = AssociationTask(config=assoc_config)
```

Comment:
This is equivalent to
```suggestion
        assoc_task = AssociationTask()
```